### PR TITLE
Update project to .NET8 and create new pins after Reset notification from ItemsSource

### PR DESCRIPTION
**Pull request by brakonier946**

### CODE REVIEW

File: lib/Maui.GoogleMaps/Maui.GoogleMaps.csproj [hunk 1]
```diff
@@ -1,21 +1,33 @@
...
 		<SingleProject>true</SingleProject>
 		<ImplicitUsings>enable</ImplicitUsings>
 		<SupportedOSPlatformVersion Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'ios'">14.2</SupportedOSPlatformVersion>
 		<SupportedOSPlatformVersion Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'android'">21.0</SupportedOSPlatformVersion>
 	</PropertyGroup>
 
-	<ItemGroup Condition="'$(TargetFramework)' == 'net7.0-android'">
+	<PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|net7.0-ios|AnyCPU'">
+		<CreatePackage>false</CreatePackage>
+	</PropertyGroup>
```
Comment (PavloLukianets) [inline lib/Maui.GoogleMaps/Maui.GoogleMaps.csproj]: please revert this change with < CreatePackage >

File: lib/Maui.GoogleMaps/Maui.GoogleMaps.csproj [hunk 2]
```diff
@@ -1,21 +1,33 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
 	<PropertyGroup>
-		<TargetFrameworks>net7.0-android;net7.0-ios</TargetFrameworks>
+		<TargetFrameworks>net8.0-android;net8.0-ios</TargetFrameworks>
 		<UseMaui>true</UseMaui>
+		<UseMauiEssentials>true</UseMauiEssentials>
```
Comment (PavloLukianets) [inline lib/Maui.GoogleMaps/Maui.GoogleMaps.csproj]: Do we need them, though?

File: lib/Maui.GoogleMaps/Maui.GoogleMaps.csproj [hunk 3]
```diff
@@ -1,21 +1,33 @@
...
+	<PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|net8.0-ios|AnyCPU'">
+		<CreatePackage>false</CreatePackage>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<PackageReference Include="Microsoft.Maui.Controls" Version="8.0.3" />
+	</ItemGroup>
+
+	<ItemGroup Condition="'$(TargetFramework)' == 'net8.0-android'">
 	  <PackageReference Include="Xamarin.GooglePlayServices.Maps" Version="118.1.0.1" />
```
Comment (PavloLukianets) [inline lib/Maui.GoogleMaps/Maui.GoogleMaps.csproj]: @brakonier946 can u also please update the packages? I guess there is a new version for android

File: lib/Maui.GoogleMaps/Maui.GoogleMaps.csproj (hunk 2 above)
Comment (brakonier946) [reply lib/Maui.GoogleMaps/Maui.GoogleMaps.csproj]: nope. removed
